pandunia-lekse.csv

```diff
@@ -1219,41 +1223,49 @@ jaze||||jazz||jazz||||||||||||||jazz (jatsi)|jazz, dżez
 jekogeta|||zho:借 (jiè), yue:借 (ze3), jpn:借 (shaku), kor:적 (jeok) + eng:check|borrow|empruntre|pedir prestado|pedir emprestado|||借入|借りる||||||||||pożyczać (pożyczyć)
 jekolekse||||loanword||préstamo lingüístico|||||||||||||||zapożyczenie (wyraz obcy)
 jele|||eng:gel, fra:gelée, rus:желе (žele), tur:jel, hin:जेल (jel), kor:젤 (jel), zho:啫哩 (zhělī)|gel (jelly)||gel (jalea, gelatina)|||||||||||||ĝelo|hyytelö|żel, galareta
-jelosi||||jealous||celoso (envidioso)||||||||||||||kade (kateellinen)|zazdrosny
+jelosi|||eng:jealous, spa:celoso, fra:jaloux|jealous||celoso (envidioso)||||||||||||||kade (kateellinen)|zazdrosny
 jena||||give birth (beget, generate)||dar a luz (enngendrar, parir)|||||||||||||naskigi|synnyttää|urodzić, rodzić; rozmnożyć się, rozmnażać się, spłodzić, płodzić
 jene|||hin:जन्म (janma), mal:ജനനം (jananaṃ), ell: γέννα (génna), eng:genesis, fra:genèse|birth||nacimiento (origen)|||||||||||||nasko|synty (syntymä)|narodziny
 jener||||parent (mother or father)||parente (madre o padre)||||||||||||||vanhempi (isä tai äiti)|rodzic (matka lub ojciec)
 jenge|||zho:爭 (zhēng), vie:tranh, kor:전쟁 (jeonjaeng), tur:kur:ceng, fas:urd:(jang), hin:जंग (jang)|fight (war, battle, combat)||pelea (guerra, batalla)|||||戦闘 (格闘)||||||||milito (batalo)|taistelu (sota)|walka, wojna
 jenger||||warrior (fighter)||luchador (guerrero)||||||||||||||soturi (taistelija)|wojownik, bojownik
 jengi||||military (martial, warlike)||macrial (militar)|||||||||||||milita|sotilaallinen|militarny, wojskowy, wojenny
-jengoxute|||zho:武术 (wǔshù), jpn:武術 (bujutsu), vie:võ thuật, kor:무술 (musul), eng:martial art|martial art||arte marcial||||||||||||||kamppailulaji|sztuka walki
+jengxute|||zho:武术 (wǔshù), jpn:武術 (bujutsu), vie:võ thuật, kor:무술 (musul), eng:martial art|martial art||arte marcial||||||||||||||kamppailulaji|sztuka walki
 jeni||||raw (crude)||crudo||сырой||生的|生 (未熟)||||||||||
 jeni baxe||||native language (mother tongue)|langue maternelle|lengua materna|língua materna|родной язык||母语||||मातृभाषा|||||hejmlingvo|äidinkieli|język ojczysty (język rodzimy)
-jenoden||||birthday||cumpleaños|||||||||||||naskiĝtago|syntymäpäivä|urodziny
-jenogrupe||||generation (cohort)||generación (cohorte)||поколение|||||||||||||
-jenoklase||||caste||casta||||||||||||||sääty (kasti)|kasta
-jenotipe||||genotype|génotype|genotipo|genótipo|генотип||基因型|遺伝子型|||||||||perimä (genotyyppi)|genotyp
-jentili||||kind (nice, gentle)||amable (atento, tierno, dulce)|||||||||||||bonkora|kiltti|uprzejmy, miły
+jenden||||birthday||cumpleaños|||||||||||||naskiĝtago|syntymäpäivä|urodziny
+jengrupe||||generation (cohort)||generación (cohorte)||поколение|||||||||||||
+jenklase||||caste||casta||||||||||||||sääty (kasti)|kasta
+jentipe||||genotype|génotype|genotipo|genótipo|генотип||基因型|遺伝子型|||||||||perimä (genotyyppi)|genotyp
+jentili|||eng:gentle, por:fra:gentil, ita:gentile|kind (nice, gentle)||amable (atento, tierno, dulce)|||||||||||||bonkora|kiltti|uprzejmy, miły
```

Comment:
"jen-tili" looks like a compound, doesn't it? This is a fine example why we should put some effort on replacing long roots like this with shorter ones or with compound words (like Spanish "amable"!).